### ISSUE TITLE
Storage not working on android 10+

### ISSUE THREAD

**Issue by rnayabed** [gluonhq/attach#232]
Storage API in Android 10+ has been changed. Adding `android:requestLegacyExternalStorage=true` flag to manifest fixed this for Android 10. However this flag is now ignored from Android 11 onwards. 

Read [here](https://stackoverflow.com/a/63365276) for more info

**Comment by jperedadnr** [comment]
@rnayabed Thanks for the info. 
Can you describe your use case? I take private storage works fine, but you are trying to access external storage, with [StorageService::getPublicStorage](https://github.com/gluonhq/attach/blob/master/modules/storage/src/main/java/com/gluonhq/attach/storage/StorageService.java#L99) for a given directory and that fails for you? What is the error you get? What directories are you trying to reach?


Can you run:

```
StorageService.create().ifPresent(s -> {
            s.getPrivateStorage().ifPresentOrElse(sp -> System.out.println("PrivateStorage = " + sp), () -> System.out.println("PrivateStorage: null "));
            s.getPublicStorage("Pictures").ifPresentOrElse(sp -> System.out.println("Pictures = " + sp), () -> System.out.println("Pictures: null "));
            System.out.println("ExternalStorageWritable = " + s.isExternalStorageWritable());;
            System.out.println("ExternalStorageReadable = " + s.isExternalStorageReadable());;
        });
```
and post what you get?

With my Android 10 (and no `requestLegacyExternalStorage` flag included, with plugin 0.1.38, and targetSdkVersion 29 (or even 30), I get:

```
PrivateStorage = /data/user/0/com.gluonhq.sample/.gluon
Pictures = /storage/emulated/0/Pictures
ExternalStorageWritable = true
ExternalStorageReadable = true
```

**Comment by rnayabed** [comment]
Previously i was just using normal java file and hardcoded path names on android. After using gluon storage api it works fine :D
